samples/v1.6/Elements/Carousel.Header.json

```diff
@@ -0,0 +1,69 @@
+ {
```

Comment:
given that we're being prompted to make this change because folks want to be able to apply `$when` to `Carousel` elements, we should probably have at least one sample making use of templating

for bonus points: make a templated sample that populates carousel pages from stuff in data

Reply:
Just added a few more test cases - let me know what you think!

Reply:
@anna-dingler, can you take a stab at adding the cards?
You can use https://github.com/microsoft/AdaptiveCards/tree/main/samples/Templates/Scenarios, and you can add both scenario card and corresponding data in there.

Reply:
@jwoo-msft Do you mean in addition to the test cases I added to `samples/v1.6/Elements`?